test/integration/suite_test.go

```diff
@@ -29,6 +29,11 @@ import (
 	"github.com/kong/kubernetes-ingress-controller/internal/manager"
 )
 
+const (
+	EnterpriseImageRepo = "kong/kong-gateway"
+	EnterpriseImageTag  = "2.5.0.0-alpine"
+)
```

Comment:
minor thing, but generally public symbols should have a godoc comment, but these symbols don't look like they need to be public anyway (at least not yet):

```suggestion
const (
	enterpriseImageRepo = "kong/kong-gateway"
	enterpriseImageTag  = "2.5.0.0-alpine"
)
```

Reply:
leave as it is atm.